docs/output.md

```diff
@@ -208,11 +223,11 @@ BAM files are further processed with [SAMtools](http://samtools.sourceforge.net/
 * `nanopore/<CALLER>/pangolin/`
   * `*.pangolin.csv`: Lineage analysis results from Pangolin.
 
-> **NB:** The value of `<CALLER>` in the output directory name above is determined by the `artic_minion_caller` parameter (Default: 'nanopolish').
+**NB:** The value of `<CALLER>` in the output directory name above is determined by the `--artic_minion_caller` parameter (Default: 'nanopolish').
 
 </details>
 
-Lineage analysis with [`Pangolin`](https://github.com/cov-lineages/pangolin)
+Phylogenetic Assignment of Named Global Outbreak LINeages ([`Pangolin`](https://github.com/cov-lineages/pangolin)) has been used extensively during the COVID-19 pandemic in order to to assign lineages to SARS-CoV-2 genome sequenced samples. A [web application](https://pangolin.cog-uk.io/) also exists that allows users to upload genome sequences via a web browser to assign lineages to genome sequences of SARS-CoV-2, view descriptive characteristics of the assigned lineage(s), view the placement of the lineage in a phylogeny of global samples, view the temporal and geographic distribution of the assigned lineage(s).

```

Comment:
`pandemic in order to to assign lineages to` &rarr; `pandemic to assign lineages to`

  and

`global samples, view the` &rarr; `global samples, and view the`